component-lib/snackbar.html

```diff
@@ -0,0 +1,294 @@
+							message that need user action .
+						</div>
+					</div>
+					<div class="component-example">
+						<div class="title">Example</div>
+						<div class="sub-title"></div>
+
+						<div class="component-display-container">
+							<div class="snackbar snackbar-primary">
+								This is primary
+								<button class="snackbar-action">
+									<i class="fa fa-times fa-lg" aria-hidden="true"></i>
+								</button>
+							</div>
+							<div class="snackbar snackbar-secondary">
+								This is secondary alert
+								<button class="snackbar-action">
+									<i class="fa fa-times fa-lg" aria-hidden="true"></i>
+								</button>
+							</div>
+							<div class="snackbar snackbar-danger">
+								Can't send photo .
+								<button class="snackbar-action">
+									<i class="fa fa-times fa-lg" aria-hidden="true"></i>
+								</button>
+							</div>
+							<div class="snackbar snackbar-success">
+								Photo sent successfully
+								<button class="snackbar-action">
+									<i class="fa fa-times fa-lg" aria-hidden="true"></i>
+								</button>
+							</div>
+						</div>
+						<div class="code-snippet-display">
+							<code>
+								<iframe
+									src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=solarized+dark&wt=sharp&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522alert%2520alert-primary%2522%253E%250A%2509%253Cp%253EThis%2520is%2520primary%2520alert.%253C%252Fp%253E%250A%2509%253Ci%2520class%253D%2522fas%2520fa-times%2520fa-lg%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-secondary%2522%253E%250A%2509%253Cp%253EThis%2520is%2520secondary%2520alert.%253C%252Fp%253E%250A%2509%253Ci%2520class%253D%2522fas%2520fa-times%2520fa-lg%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-danger%2522%253E%250A%2509%253Cdiv%2520class%253D%2522alert-sub-container%2522%253E%250A%2509%2509%253Ci%2520class%253D%2522fas%2520fa-exclamation-triangle%2522%253E%253C%252Fi%253E%250A%2509%2509%253Cp%253EThis%2520is%2520danger%2520alert.%253C%252Fp%253E%250A%2509%253C%252Fdiv%253E%250A%2509%253Ci%2520class%253D%2522fas%2520fa-times%2520fa-lg%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-success%2522%253E%250A%2509%253Cdiv%2520class%253D%2522alert-sub-container%2522%253E%250A%2509%2509%253Ci%2520class%253D%2522far%2520fa-check-circle%2520fa-lg%2522%253E%253C%252Fi%253E%250A%2509%2509%253Cp%253EThis%2520is%2520success%2520alert.%253C%252Fp%253E%250A%2509%253C%252Fdiv%253E%250A%2509%253Ci%2520class%253D%2522fas%2520fa-times%2520fa-lg%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+									style="
+										width: 680px;
+										height: 480px;
+										border: 0;
+										transform: scale(1);
+										overflow: hidden;
+									"
```

Comment:
You could move the inline styling of iframe to the css file

Reply:
Rest looks good to merge!

Reply:
Sure. I will change.